tests/integration/targets/x509_crl/tests/validate.yml

```diff
@@ -102,3 +102,11 @@
           ['commonName', 'CRL'],
           ['commonName', 'Test'],
         ]
+
+- name: Validate CRL 3 info
+  assert:
+    that:
+      - crl_3.revoked_certificates == crl_3_info.revoked_certificates
+      - crl_3.revoked_certificates[0].issuer == [
+          "DNS:ca.example.org",
+        ]
```

Comment:
I was surprised that this plain scalar worked multi-line without `>` or `>-` or something... I did find at least one yaml parser online that didn't like it: https://codebeautify.org/yaml-validator/y22bd59f9

But it appears that validator is wrong; and it clearly works in Ansible regardless, but you may want to change it if only for consistency with the other asserts in the file.

Or not 🤷

Reply:
According to the YAML definition this should be fine: https://yaml.org/spec/1.2.2/#plain-style
I'd probably rather remove the `>` from the other blocks in this file (but not in this PR) :)